ztp/policygenerator/resources/acm-policy-du-platform-check-template.yaml

```diff
@@ -0,0 +1,185 @@
+                  updatedMachineCount: 1
+                  unavailableMachineCount: 0
+                  conditions:
+                    - type: Updated
+                      status: "True"
+                    - type: Updating
+                      status: "False"
+                  configuration:
+                    source:
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: container-mount-namespace-and-kubelet-conf
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: load-sctp-module
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: 02-master-workload-partitioning
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: 04-accelerated-container-startup-master
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: 05-chronyd-dynamic-master
+                      - apiVersion: machineconfiguration.openshift.io/v1
+                        kind: MachineConfig
+                        name: 50-performance-openshift-node-performance-profile
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: apps/v1
+                kind: Deployment
+                metadata:
+                  name: sriov-network-operator
+                  namespace: openshift-sriov-network-operator
+                status:
+                  conditions:
+                    - type: Available
+                      status: "True"
+                      reason: MinimumReplicasAvailable
+                  availableReplicas: 1
+                  readyReplicas: 1
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: sriovnetwork.openshift.io/v1
+                kind: SriovNetworkNodeState
+                metadata:
+                  namespace: openshift-sriov-network-operator
+                status:
+                  syncStatus: Succeeded
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: apps/v1
+                kind: Deployment
+                metadata:
+                  name: ptp-operator
+                  namespace: openshift-ptp
+                status:
+                  conditions:
+                    - type: Available
+                      status: "True"
+                      reason: MinimumReplicasAvailable
+                  availableReplicas: 1
+                  readyReplicas: 1
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: apps/v1
+                kind: Deployment
```

Comment:
We should also verify that the PerformanceProfile status includes Available=true and Degraded=false. This can be done in a subsequent improvement.